### DISCUSSION TITLE
Transitioning from previous version to new version of QSIprep.

### DISCUSSION THREAD

**Discussion by dcossio** [PennLINC/qsiprep#445] [General]
Hi,

A previous lab member of our preprocessed using version 0.4.5 , one specific argument used was the --force-spatial-normalization. I am attempting to use the most recent version instead of 0.4.5 to preprocess data, but I have noticed that the force-spatial-normalization is no longer an argument or may have been changed to something else ? Does anyone know what argument I would use to replace it? 

-DC

**Comment by smeisler** [comment]
Hi, there is no replacement for `--force-spatial-normalization`, as all outputs are now in native space (albeit aligned to AC-PC line).

**Reply by mattcieslak** [reply]
It will also do a spatial normalization to MNI2009cAsym by default now (no flag necessary)

**Reply by smeisler** [reply]
Yes, sorry, meant to specify that DWI outputs are only in native space, my bad!
